### PR TITLE
[travis] Add flambda testing.

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -103,6 +104,21 @@ matrix:
...
           packages: *extra-packages
 
+    # Full test-suite with flambda
+    - env:
+      - TEST_TARGET="test-suite"
+      - COMPILER="4.05.0+flambda"
+      - FINDLIB_VER="1.7.3"
+      - CAMLP5_VER="7.01"
+      - NATIVE_COMP="no"
+      - EXTRA_CONF="-coqide opt -with-doc yes -flambda-opts -O3"
```
Comment (maximedenes) [inline .travis.yml]: Do you really want to build the doc? Looks unrelated to `flambda` doesn't it?

Comment (maximedenes) [reply .travis.yml]: But what is the motivation? Do you have a scenario in mind where `flambda` could interfere with the LaTeX compilation?

Comment (ejgallego) [reply .travis.yml]: The motivation is to do a full build with a new switch. Whether that is a good motivation, I don't know, but I like to test compiler switches in full. The same could be said for the `4.05.0` CI target.

Comment (maximedenes) [reply .travis.yml]: Ok, I'm fine either way. The only reason why I'm mentioning this is that our Travis resources sometimes feel limited.
